sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1106,6 +1106,22 @@ object functions {
   def log1p(columnName: String): Column = log1p(Column(columnName))
 
   /**
+   * Computes the logarithm of the given value in Base 2.
```

Comment:
because we use upper case Base for `log10` also, I just try to keep consistence.
should I modify it to lower case also?